test/e2e/dynamic_provisioning_test.go

```diff
@@ -290,80 +288,80 @@ func (t *dynamicProvisioningTestSuite) defineTests(isMultiZone bool, schedulerNa
 
 	// Track issue https://github.com/kubernetes/kubernetes/issues/70505
 	ginkgo.It(fmt.Sprintf("should create a volume on demand and mount it as readOnly in a pod [kubernetes.io/azure-disk] [disk.csi.azure.com] [Windows] [%s]", schedulerName), func() {
-		pods := []testsuites.PodDetails{
+		pods := []testtypes.PodDetails{
 			{
-				Cmd: convertToPowershellorCmdCommandIfNecessary("touch /mnt/test-1/data"),
-				Volumes: t.normalizeVolumes([]testsuites.VolumeDetails{
+				Cmd: testutil.ConvertToPowershellorCmdCommandIfNecessary("touch /mnt/test-1/data"),
+				Volumes: testutil.NormalizeVolumes([]testtypes.VolumeDetails{
 					{
 						FSType:    "ext4",
 						ClaimSize: "10Gi",
-						VolumeMount: testsuites.VolumeMountDetails{
+						VolumeMount: testtypes.VolumeMountDetails{
 							NameGenerate:      "test-volume-",
 							MountPathGenerate: "/mnt/test-",
 							ReadOnly:          true,
 						},
 					},
-				}, isMultiZone),
-				IsWindows: isWindowsCluster,
+				}, []string{}, isMultiZone),
```

Comment:
Shouldn't you pass `t.allowedTopologyValues` here and everywhere `testutil.NormalizeVolume` is used instead of `[]string{}`?

Reply:
Fixed